tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -178,6 +178,10 @@ bgp/test_bgpmon.py:
     conditions:
       - "'backend' in topo_name or 't2' in topo_name"
 
+bgp/test_bgpmon_v6.py::test_bgpmon_no_ipv6_resolve_via_default:
+  skip:
+    reason: "Known issue with no ipv6 nht resolve-via-default not preventing connection to BGP neighbor"
+
```

Comment:
can we update comment that in passive mode this test is not applicable

Reply:
Updated